LATEST

```diff
@@ -1,4 +1,4 @@
-1c18058f019229cd0af64669af0de31d0cec916d 1.2.0-snapshot.20200602.4310.0.1c18058f
+1386abc05ac894941f7e6eccf7e0721252957243 1.2.0-snapshot.20200603.4345.0.1386abc0
```

Comment:
This should be 1.3. Keep the 1.2 line and add a new line for that.

Reply:
Do we already have a 1.2.0?

Reply:
No but we have frozen the RC, so unless someone comes up with a blocking issue, 4310 will be 1.2.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6207/commits/9b23c78910cf6c844baf1950c202a89525e98e4b